__tests__/routes/profiles.test.js

```diff
@@ -153,8 +153,8 @@ describe('Profile Router', () => {
         expect(authRequired).toBeCalled();
       });
 
-      it('responds with status 201', async () => {
-        const expected = 201;
+      it('responds with status 200', async () => {
```

Comment:
Interesting - this is a case of changing the test to reflect a mistake in our code, rather than changing our code to reflect the intended response as indicated by our test.

Status code 201 indicates that a request was successful and as a result, a resource has been created, which exactly what this endpoint - built to post new users to our database - is doing. as such, it would have been a better option to update our code to match this expectation, rather than change our code to match the bad practices currently represented in this actual endpoint.